PhenotypeOrthologous.pm

```diff
@@ -0,0 +1,191 @@
+Copyright [2016-2023] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 CONTACT
+
+ Ensembl <http://www.ensembl.org/info/about/contact/index.html>
+
+=cut
+
+=head1 NAME
+
+PhenotypeOrthologous
+
+=head1 SYNOPSIS
+
+ mv PhenotypeOrthologous.pm ~/.vep/Plugins
+ ./vep -i variations.vcf --plugin PhenotypeOrthologous,file=rat_human.gff3.gz,species=rat
+
+=head1 DESCRIPTION
+
+ A VEP plugin that retrieves phenotype information associated with orthologous genes from from model organisms.
+
+ The plugin presently only supports two model organisms, rats and mouse. 
+
+ The PhenotypeOrthologous file can be downloaded from https://ftp.ensembl.org/pub/current_variation/PhenotypeOrthologous
+
+ The plugin can be run: 
+   
+  ./vep -i variations.vcf --plugin PhenotypeOrthologous,file=rat_mouse_orthologous.gff3.gz
+ 
+  To return only results for rat :
+    ./vep -i variations.vcf --plugin PhenotypeOrthologous,file=rat_mouse_orthologous.gff3.gz,species=rat
+
+    ./vep -i variations.vcf --plugin PhenotypeOrthologous,file=rat_mouse_orthologous.gff3.gz,species=rattus_norvegicus
+  To return only results for mouse:
+    ./vep -i variations.vcf --plugin PhenotypeOrthologous,file=rat_mouse_orthologous.gff3.gz,species=mouse
+
+    ./vep -i variations.vcf --plugin PhenotypeOrthologous,file=rat_mouse_orthologous.gff3.gz,species=mus_muculus
+
+ The tabix utility must be installed in your path to use this plugin.
+ Check https://github.com/samtools/htslib.git for instructions.
+
+=cut 
+
+package PhenotypeOrthologous;
+
+use strict;
+use warnings;
+
+use Bio::EnsEMBL::Variation::Utils::BaseVepTabixPlugin;
+
+use base qw(Bio::EnsEMBL::Variation::Utils::BaseVepTabixPlugin);
+
+sub new {
+  my $class = shift;
+
+  my $self = $class->SUPER::new(@_);
+ 
+  $self->expand_left(0);
+  $self->expand_right(0);
+
+  $self->get_user_params();
+    
+  my $params = $self->params_to_hash();
+
+  my $file = $params->{file};  
+  my $species = $params->{species};
+
+  die "File needs to be specified to run the PhenotypeOrthologus plugin. \n" if  (!$file);
+  $self->add_file($file);
+
+  #die "Species needs to be specified to run the PhenotypeOrthologous plugin. \n" if (!$species); 
+
+  if ( defined($species) && $species ne "rattus_norvegicus" && $species ne "rat" && $species ne "mouse" && $species ne "mus_muculus") {
+    die "Phenotype Orthologous plugin only works for rat and mouse \n"
```

Comment:
Phenotype Orthologous -> PhenotypeOrthologous
Also typo in mus_musculus in the previous line.